js/ui/diagram/ui.diagram.commands.js

```diff
@@ -99,6 +102,33 @@ const DiagramCommands = {
                 displayExpr: "name",
                 valueExpr: "value"
             },
+            {
+                widget: "dxButton",
+                icon: "export",
+                items: [
+                    {
+                        command: DiagramCommand.ExportSvg, // eslint-disable-line spellcheck/spell-checker
+                        text: "Export to SVG",
```

Comment:
Possible, before release 19.1 you will should use DevExtreme's localization mechanism

Reply:
ok